files/management/commands/fix_media_paths.py

```
@@ -1,236 +1,160 @@
+                total_remaining += self._verify_field(model, field_name, media_root)
+
+            if total_remaining == 0:
+                self.stdout.write(self.style.SUCCESS('  All fields clean — no absolute paths remain.'))
 
         # Summary
-        self.stdout.write(self.style.SUCCESS('\n=== Summary ==='))
-        self.stdout.write(f'Media paths fixed: {media_fixed} (errors: {media_errors})')
-        self.stdout.write(f'Encoding paths fixed: {encoding_fixed} (errors: {encoding_errors})')
-        self.stdout.write(f'Total fixed: {media_fixed + encoding_fixed}')
-        self.stdout.write(f'Total errors: {media_errors + encoding_errors}')
+        self.stdout.write(self.style.SUCCESS(f'\n=== Summary ==='))
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)